style.css

```diff
@@ -0,0 +1,170 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  position: relative;
+}
+
+.body {
+  width: 100vw;
+  min-height: 100vh;
+  position: relative;
+}
+
+header {
+  position: sticky;
+  top: 0;
+}
+
+nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 30px;
+  background-color: #3c3a39;
+}
+
+.logo,
+.menu {
+  height: 24px;
+  cursor: pointer;
+}
+
+.flex {
+  display: flex;
+}
+
+.header_content {
+  padding: 0 8%;
```

Comment:
Please stick to the design and fix the left and right padding here. As I mentioned earlier, you need to stick to the design as much as possible.

<img width="472" alt="Screenshot 2023-01-16 at 11 36 05 PM" src="https://user-images.githubusercontent.com/25887253/212762189-9586fe47-1c62-4e0b-9946-9646fde7c527.png">
<img width="940" alt="Screenshot 2023-01-16 at 11 36 28 PM" src="https://user-images.githubusercontent.com/25887253/212762190-0947b0be-1864-4476-8d9c-001a78704e4a.png">